lib/rubocop/cop/rspec/repeated_block_description.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Check for repeated describe and context block descriptions.
+      #
+      # @example
+      #
+      #    # bad
+      #    describe 'cool feature' do
+      #      # example group
+      #    end
+      #
+      #    describe 'cool feature' do
+      #      # example group
+      #    end
+      #
+      #    # bad
+      #    context 'when case x' do
+      #      # example group
+      #    end
+      #
+      #    describe 'when case x' do
+      #      # example group
+      #    end
+      #
+      #    # good
+      #    describe 'cool feature' do
+      #      # example group
+      #    end
+      #
+      #    describe 'another cool feature' do
+      #      # example group
+      #    end
+      #
+      #    # good
+      #    context 'when case x' do
+      #      # example group
+      #    end
+      #
+      #    describe 'when another case' do
+      #      # example group
+      #    end
+      #
```

Comment:
Can you please add an example for several similar example groups defined in the same context (most frequently seen in root context actually) with an additional docstring?
```ruby
RSpec.describe 'A', 'dog' do
  # ...
end
RSpec.describe 'A', 'cat' do
  # ...
end
```